manifest_staging/deployment/provider-azure-installer-windows.yaml

```
@@ -46,6 +46,10 @@ spec:
             limits:
               cpu: 100m
               memory: 200Mi
+          ports:
+            - containerPort: {{ .Values.metricsAddr }}
```

```suggestion
            - containerPort: 8898
```

This is a static yaml